integration_test/run_complete.R

```diff
@@ -1,12 +1,25 @@
-# Integration test running all of our methods
+#!/usr/bin/env Rscript
 
+# Integration test running all of our methods
 library(devtools)
 devtools::load_all()
 
+get_env <- function(variable, default) {
+  if (Sys.getenv(variable) == "") {
+    return(default)
+  }
+  return(Sys.getenv(variable))
+}
+
+username <- get_env("AVATAR_USERNAME", default = "user_integration")
+password <- get_env("AVATAR_PASSWORD", default = "password_integration")
+base_url <- get_env("AVATAR_BASE_URL", default = "http://localhost:8000")
```

Comment:
This is a good refacto.
As we're on those lines of code: I wouldn't put default here, because a user could be tempted to use it like a quickstart script and wouldn't get understandable error. But I'd put the env params in the Makefile, as `make test-integration` is what is running this.